sw/device/lib/runtime/otbn.c

```diff
@@ -195,3 +196,16 @@ otbn_result_t otbn_zero_data_memory(otbn_t *ctx) {
   }
   return retval;
 }
+
+void otbn_dump_dmem(const otbn_t *ctx, uint32_t max_addr) {
```

Comment:
For consistency, please null-check `ctx` and return an error.

Reply:
Good catch, added.